tests/main/snap-repair/uc20-recover.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+# show what mode we are in
+echo "$SNAP_SYSTEM_MODE"
```

Comment:
are these echo checked?

Reply:
mmm they were supposed to be checked, but I don't seem to actually check them anywhere, I will add a snippet to the test to check that is correct

Reply:
this is checked now